package.json

```diff
@@ -7,7 +7,7 @@
     "watch-js": "webpack --watch",
     "watch-scss": "watch -p 'static/sass/**/*.scss' -c 'yarn run build-scss'",
     "build": "yarn run build-css && yarn run build-js",
-    "build-css": "node-sass --include-path node_modules static/sass --output-style compressed --output static/css && postcss --use autoprefixer --no-map --replace 'static/css/**/*.css'",
+    "build-css": "sass static/sass/styles.scss static/css/styles.css --load-path=node_modules --style=compressed && postcss --map false --use autoprefixer --replace 'static/css/**/*.css'",
```

Comment:
I think you want to keep `--no-map` to remove inline source maps.

Reply:
I only copy-pasted this from charmhub. I know very little about these flags so I looked into it now because I thought the `--map false` should achieve the same as `--no-map`.

From reading.. `--map false` should achieve the same, but it still generates the file.

`--no-map` seems to work, so I will put that back and get rid of `--map false`

Reply:
Thanks. I believe the difference is that `--no-map` disables inline source maps which is a huge blob at the end of the generated CSS file. Whereas, `--map false` disables the separate map file from being generated.

Reply:
I looked into it a bit more. `--no-map` or  `--map false` flags for `postcss` is not preventing the map file from being generated. Adding `--no-source-map` for `sass` did the job. I will remove `postcss` map flags unless you think it's a bad idea.

Reply:
We want `--no-map` to prevent the inline source maps. 

To see what I mean:
- visit the stylesheet generated by the demo: https://charmed-kubeflow-io-42.demos.haus/static/css/styles.css?v=20cbcdf
- scroll half way down and you will see there is a huge block of source map at the end. Doubling the CSS size of the site.

We don't mind the separate sourcemap file being generated as it's only downloaded when the inspector is open. But we don't want it appended to the built CSS file.

Reply:
Now I get it. So we don't mind the .map file. I will put it back.